src/SourceBuild/content/eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -18,6 +18,7 @@ resources:
       branches:
         include:
           - refs/heads/release/*.0.1xx-preview*
+          - refs/heads/release/9.0.1xx*

```

Comment:
This will run the pipeline on release/9.0.1xx-rc1. Since we have an internal branch for rc1, do we need/want to run the pipeline on the public branch as well?

Reply:
Good catch. You're right, we want to avoid that. I've updated to explicitly check the 9.0.1xx branch.